Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/testclasses/SlidesLimitSwitchTest.java

```diff
@@ -0,0 +1,39 @@
+package org.firstinspires.ftc.team6220_PowerPlay.testclasses;
+
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.team6220_PowerPlay.BaseTeleOp;
+import org.firstinspires.ftc.team6220_PowerPlay.Constants;
+
+import java.util.Random;
+
+@TeleOp(name = "SlidesLimitSwitchTest")
+public class SlidesLimitSwitchTest extends BaseTeleOp {
+        Random rng = new Random();
+        @Override
+        public void runOpMode() {
+            initialize();
+            driveSlides(0);
+            waitForStart();
+
+            while (opModeIsActive()) {
+                for(int i = 0; i <= 20; i++) {
+                    for(int j = 0; j <= 5; j++) {
+                        driveSlides(rng.nextInt(Constants.SLIDE_TOP-50)+50);
+                    }
+                    while(gamepad1.a) {
+
+                    }
+                    driveSlides(0);
+                }
+
+                for(Object o : telemetrySave) {
+                    telemetry.addData("test", o);
+                }
+                telemetry.update();
+                while(gamepad1.a && opModeIsActive()) {
+
+                }
```

Comment:
Same question. Also `opModeIsActive()` will always be true here because it's inside the larger while loop.
```suggestion
                while (gamepad1.a) { }
```

Reply:
According to the Java code convention empty loops should be formatted like `while(condition);` https://www.oracle.com/technetwork/java/codeconventions-150003.pdf#G3.460 section 7.7. I'll fix it

Reply:
Awesome! Good to know the convention.